trieste/models/model_interfaces.py

```diff
@@ -311,6 +311,43 @@ def update(self, dataset: Dataset) -> None:
 
         self.model.data = dataset.query_points, dataset.observations
 
+    def covariance_between_points(
+        self, query_points_1: TensorType, query_points_2: TensorType
+    ) -> TensorType:
+        """
+        Compute the posterior covariance between sets of query points.
+
+        .. math:: \Sigma_{12} = K_{12} - K_{x1}(K_{xx} + \sigma^2 I)^{-1}K_{x2}
+
+        :param query_points_1: Set of query points with shape [N, D]
+        :param query_points_2: Sets of query points with shape [M, D]
+
+        :return: Covariance matrix between the sets of query points with shape [N, M]
+        """
+        tf.debugging.assert_shapes([(query_points_1, ["N", "D"]), (query_points_2, ["M", "D"])])
+
+        x, _ = self.model.data
+        num_data = x.shape[0]
+        s = tf.linalg.diag(tf.fill([num_data], self.model.likelihood.variance))
+
```

Comment:
No jitter?

Reply:
The GPflow GPR model doesn't use jitter - the likelihood variance has to be sufficiently large to avoid cholesky errors.